brianlong/www.validators.app
# 183522592 - add testnet stats for homepage

#### What's this PR do?
PR fixes the testnet data for the homepage on stage and production servers.

#### How should this be manually tested?
Should be tested on stage as well. Verify the homepage stats between the testnet and mainnet on stage. On the localhost we use the testnet source for all the time, so the testing is rather difficult.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183522592)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
